src/plots/plots.js

```diff
@@ -1390,6 +1385,11 @@ plots.supplyLayoutModuleDefaults = function(layoutIn, layoutOut, fullData, trans
         Cartesian.finalizeSubplots(layoutIn, layoutOut);
     }
 
+    // sort subplot lists
+    for(var subplotType in layoutOut._subplots) {
+        layoutOut._subplots[subplotType].sort(Lib.subplotSort);
```

Comment:
So, if I get this right, PR https://github.com/plotly/plotly.js/pull/2227 (which got merged in `1.32.0`) is the culprit for https://github.com/plotly/plotly.js/issues/2443 ?

Reply:
Yes, I think so, at least that was the main part of it.